src/drawtypes/label.cpp

```diff
@@ -55,10 +55,14 @@ namespace drawtypes {
           repl.insert(0_z, tok.min - repl.length(), tok.zpad ? '0' : ' ');
         }
 
+        if (!std::binary_search(m_token_whitelist.cbegin(), m_token_whitelist.cend(), token)) {
+          repl = string_util::replace_all(repl, "%", "%%");
+        }
+
         /*
          * Only replace first occurence, so that the proper token objects can be used
          */
-        m_tokenized = string_util::replace(m_tokenized, token, move(repl));
+        m_tokenized = string_util::replace(m_tokenized, token, repl);
```

Comment:
Removed `move` since the parameter is a `const std::string&` so the `move` does nothing.